build/test/integration/distribution_test.js

```diff
@@ -150,15 +151,19 @@ suite('Distribution mechanism', function() {
       path.join(cusDir, 'mobizilla',
         'mobizilla_expected_bookmarks.json'));
   }
+  */
 
+  /*
   function validateWappush() {
     var wappushZipPath = path.join(process.cwd(), 'profile',
       'webapps', 'wappush.gaiamobile.org', 'application.zip');
 
     helper.checkFileContentByPathInZip(wappushZipPath, 'js/whitelist.json',
       path.join(cusDir, 'wappush-whitelist.json'), true);
   }
+  */
```

Comment:
WapPush will be needed (we checked this on irc yesterday) so I think I will keep it :)